docker/puppetdb/Dockerfile

```diff
@@ -18,6 +18,10 @@ COPY --from=builder /app/target/puppetdb.jar /
 ARG vcs_ref
 ARG build_date
 ARG version="5.1.5"
+# used by entrypoint to submit metrics to Google Analytics;
+# published images should use "production" for this build_arg
+ARG pupperware_analytics_stream="dev"
+ENV PUPPERWARE_ANALYTICS_STREAM="$pupperware_analytics_stream"
```

Comment:
The difference between pdb and puppetserver was because puppetserver has the parent-child image and I wanted the child image to reuse as much from the parent.

To me it seems ideal to keep as much as possible defined in the `10-analytics.sh` script, so I strived for that and added ENV/ARG values in the Dockerfile only as necessary.

They're not meant to be overridden, and each container has its own analytics.sh script so we're not trying to reuse a generic analytics.sh script (like we do for ssl.sh).